model/train_gan.py

```diff
@@ -138,17 +175,17 @@ def generate_real_samples(dataset, n_samples):
 	
     return X, y
 
-def summarize_performance(output_file, epoch, d_loss_real, d_loss_fake, g_loss,
+def summarize_performance(output_file, epoch, d_loss_real, d_loss_fake, g_loss, acc_real, acc_fake,
                           g_model, d_model, dataset, latent_dim, n_samples,
                           save_model=False):
     # prepare real samples
-    X_real, y_real = generate_real_samples(dataset, n_samples)
+    # X_real, y_real = generate_real_samples(dataset, n_samples)
     # evaluate discriminator on real examples
-    _, acc_real = d_model.evaluate(X_real, y_real, verbose=0)
+    # _, acc_real = d_model.evaluate(X_real, y_real, verbose=0)
     # prepare fake examples
     x_fake, y_fake = generate_fake_samples(g_model, latent_dim, n_samples)
     # evaluate discriminator on fake examples
-    _, acc_fake = d_model.evaluate(x_fake, y_fake, verbose=0)
+    # _, acc_fake = d_model.evaluate(x_fake, y_fake, verbose=0)
     # summarize discriminator performance
     print('Accuracy real: %.0f%%, fake: %.0f%%' %
           (acc_real * 100, acc_fake * 100))
```

Comment:
where is acc_real and acc_fake calculated now?

Reply:
from https://machinelearningmastery.com/practical-guide-to-gan-failure-modes/
they directly get the accuracy from the `train_on_batch` `_loss1, d_acc1 = d_model.train_on_batch(X_real, y_real)`

Reply:
I thought that could explain the difference in the metrics values we output